web/client/actions/featuregrid.js

```diff
@@ -177,10 +177,19 @@ export function toggleSelection(features) {
         features
     };
 }
-export function setSelectionOptions({multiselect = false} = {}) {
+/**
+ * Changes the selection options for the feature grid. It allows to set independently the possibility
+ * to do multiple selection and to show/hide the checkboxes.
+ * @memberof actions.featuregrid
+ * @param {boolean} [options.multiselect=false] if true, allows multiple feature selection
```

Comment:
should we remove it or restore it?

```suggestion
 * @param {boolean} [options.multiselect] if true, allows multiple feature selection
```